src/csv.jl

```diff
@@ -70,12 +70,12 @@ csvread(file::String, delim=','; kwargs...) = _csvread_f(file, delim; kwargs...)
 
 function csvread(file::IOStream, delim=','; kwargs...)
     mmap_data = Mmap.mmap(file)
-    _csvread(String(mmap_data), delim; kwargs...)
+    _csvread(WeakRefString(pointer(mmap_data), length(mmap_data)), delim; kwargs...)
```

Comment:
@andreasnoack Are you sure we want this back?

Reply:
No. I'm not. I just reverted the old commit. I remember timing this one and it didn't really matter so lets just keep `String`.